pom.xml

```diff
@@ -15,7 +15,7 @@
    <parent>
       <groupId>org.eclipse.hawkbit</groupId>
       <artifactId>hawkbit-parent</artifactId>
-      <version>0.3.0M2</version>
+      <version>0.3.0-SNAPSHOT</version>
```

Comment:
Keep in mind when you file the final PR that this should always point to an official version e.g. `0.3.0M3`. 
Otherwise community has to build hawkBit first to use device simulator